.travis.yml

```diff
@@ -1,24 +1,26 @@
 language: c
+git:
+  submodules: false
+
+env: B=autotools
+
 install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq curl
   - curl http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04/Release.key | sudo apt-key add -
```

Comment:
Otherwise command not found error would be found in the logs and the `apt-key add` would also fail. It may or may not be generally installed on desktop systems, but Travis is a development CI which comes pre-installed with various tools.